compose/project.py

```diff
@@ -412,10 +440,12 @@ def down(
         self.remove_images(remove_image_type)
 
     def remove_images(self, remove_image_type):
-        for service in self.get_services():
+        for service in self.services:
```

Comment:
The call to `self.get_services` was actually unnecessary before, since without any parameters it would just return `self.services`.
But now it would filter services by enabled profiles, so this was changed so `remove_images` and thus `down` would act on _all_ defined services.